config/secrets.yml

```diff
@@ -17,16 +17,18 @@
 
 # Environmental secrets are only available for that specific environment.
 
-development:
```

Comment:
@s-espinosa @icorson3 I tried to remove this, but for some reason it still got pushed to GH?